packages/chains/chains/8453.ts

```diff
@@ -2,6 +2,9 @@ import type { Chain } from "../src/types";
 export default {
   "chain": "ETH",
   "chainId": 8453,
+  "ens": {
+    "registry": "0xeCBaE6E54bAA669005b93342E5650d5886D54fc7"
+  },
```

Comment:
Added BNS data to Base chain

Reply:
unfortunately this data will get overriden next time the chains get sync'd from remote.

I think it'd be easier to have it hardcoded for now at resolution time.

next iteration we can add this to the remote chains directly